src/app/legacy/containers/Navigation/index.jsx

```diff
@@ -7,6 +7,8 @@ import {
 import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
 import useViewTracker from '#app/hooks/useViewTracker';
 import { RequestContext } from '#contexts/RequestContext';
+import isLive from '#app/lib/utilities/isLive';
+import LanguageNavigationLazy from '#app/components/LanguageNavigation/lazy';
```

Comment:
No need to include Lazy in the component name
```suggestion
import LanguageNavigation from '#app/components/LanguageNavigation/lazy';
```